pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -1757,60 +1702,23 @@ func (s *AWSCloud) updateInstanceSecurityGroupsForLoadBalancer(lb *elb.LoadBalan
 		return fmt.Errorf("Could not determine security group for load balancer: %s", orEmpty(lb.LoadBalancerName))
 	}
 
-	// Get the actual list of groups that allow ingress from the load-balancer
+	// Gather all security groups
 	describeRequest := &ec2.DescribeSecurityGroupsInput{}
-	filters := []*ec2.Filter{}
-	filters = append(filters, newEc2Filter("ip-permission.group-id", loadBalancerSecurityGroupId))
-	describeRequest.Filters = s.addFilters(filters)
```

Comment:
I think we need these filters, otherwise we are acting on security groups that aren't part of our cluster.